src/m_config.c

```diff
@@ -2502,6 +2507,10 @@ char *M_GetAutoloadDir(const char *iwadname)
     {
         char *prefdir;
         prefdir = SDL_GetPrefPath("", PACKAGE_TARNAME);
+        if (prefdir == NULL)
+        {
+            I_Error("M_GetAutoloadDir: SDL_GetPrefPath failed");
```

Comment:
I think we can also live without an autoload directory. Could you somehow make this non-fatal as well?

Reply:
Yes but this will require fixes in several places, also in Heretic and Hexen. I do not use this feature so I can't test if it still works correctly.

Reply:
But your will to block others that could test? You seemed to find a technique to test above? More like you don't want to be helpful, rather than you can't.

Reply:
@fabiangreffrath Done. All the games now start when prefpath is `NULL` but there are a lot of warnings. Normal startup still works as before. I do not use autoload so I can't test if it works or behaves strangely.